[2023-10-07T02:13:53.034+0000] {taskinstance.py:1147} INFO - Dependencies not met for <TaskInstance: read_db_by_plain_python.read_db manual__2023-10-07T01:51:59.701486+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2023-10-07T02:13:53.043+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
